packages/opa-react/src/use-authz.ts

```diff
@@ -26,28 +26,35 @@ export default function useAuthz(
   if (context === undefined) {
     throw Error("Authz/useAuthz can only be used inside an AuthzProvider");
   }
-  const { defaultPath, defaultInput, defaultFromResult, queryClient } = context;
+  const { defaultPath, defaultInput, defaultFromResult, queryClient, sdk } =
+    context;
   const p = path ?? defaultPath;
   const i = mergeInput(input, defaultInput);
   const fromR = fromResult ?? defaultFromResult;
 
-  const {
-    // NOTE(sr): we're ignoring 'status'
-    data: result,
-    error,
-    isFetching: isLoading,
-  } = useQuery<Result>(
-    {
-      queryKey: [p, i],
-      meta: { fromResult: fromR },
-    },
-    queryClient,
-  );
-  return {
-    isLoading,
-    result,
-    error: error != null ? error : undefined,
-  } as UseAuthzResult<Result>;
+  try {
+    console.log({ p, i });
+    const {
+      // NOTE(sr): we're ignoring 'status'
+      data: result,
+      error,
+      isFetching: isLoading,
+    } = useQuery<Result>(
+      {
+        queryKey: [p, i],
+        meta: { fromResult: fromR },
+      },
+      queryClient,
+    );
+    return {
+      isLoading,
+      result,
+      error: error != null ? error : undefined,
+    } as UseAuthzResult<Result>;
+  } catch (e) {
+    console.error(e);
```

Comment:
I've only added this because I don't know what happened to that one request. It didn't help, though.